src/components/Cell/index.stories.tsx

```diff
@@ -1,6 +1,14 @@
+/*
+ * @Author: zhaohui
+ * @Date: 2021-05-12 18:14:09
+ * @LastEditTime: 2021-05-13 19:34:24
+ * @LastEditors: zhaohui
+ * @Description:
+ * @FilePath: /vant-react/src/components/Cell/index.stories.tsx
+ */
 import React from 'react';
 
-import Cell from '.';
```

Comment:
renaming this file could be a breaking change for some projects already using vant. I would be more comfortable merging this if you could keep Cell as the index.tsx, and export group as an optional exported module